store/src/java/com/zimbra/cs/datasource/imap/ImapFolderSync.java

```diff
@@ -277,6 +279,10 @@ public void syncMessages() throws ServiceException, IOException {
 
         // Restore previously cached sync state or create new state
         syncState = imapSync.removeSyncState(localFolder.getId());
+        int lastChangeId = 0;
+        if (remoteFolder.getPath().equals(DRAFTS) && syncState != null) {
```

Comment:
Can we keep it like this if (syncState != null && remoteFolder.getPath().equals(DRAFTS)),
so that null check fails, there will no need to call getPath() method